code/__HELPERS/type2type.dm

```diff
@@ -39,6 +39,34 @@
 		else
 	return
 
+//Turns a direction into a nautical direction
+/proc/dir2nautical(direction)
+	switch(direction)
+		if(NORTH)
+			return "fore"
+		if(SOUTH)
+			return "aft"
+		if(EAST)
+			return "starboard"
+		if(WEST)
+			return "port"
+		else
+	return
+
+//Turns a nautical direction into a direction
+/proc/nautical2dir(direction)
+	switch(uppertext(direction))
+		if("FORE")
+			return NORTH
+		if("AFT")
+			return SOUTH
+		if("STARBOARD")
+			return EAST
+		if("PORT")
+			return WEST
+		else
```

Comment:
else... what?

Reply:
I was following the other procs above it. Rather it have an else return null, or nothing at all, then?

Reply:
It should CRASH

Reply:
Done.